docker-entrypoint.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
-python manage.py migrate  # Apply database migrations
+
+# The postgres container actually takes a little while to start up,
+# and many times the django container finishes starting up before the
+# postgres container does. If this happens, the django container will
+# error out the first time it tries to access the postgres container.
+# To avoid this, we will wait until the "python manage.py migrate",
+# command to apply database migrations, runs.
+# For more information, see: https://docs.docker.com/compose/startup-order/
+until python manage.py migrate; do
+  >&2 echo "Postgres is unavailable - sleeping"
+  sleep 1
+done
```

Comment:
Cool! It's nice to have a way to do this. I'm a little surprised that Docker doesn't do this more gracefully in `docker-compose`, actually, given that we have a `depends_on` configuration in the `core` section of our `docker-compose.yml` file. Maybe it's a "not implemented yet" kind of feature.

Reply:
Right, I agree - I thought docker-compose would handle that more gracefully, especially, as you point out, as the `core` `depends_on` the database container.